src/styles/cards.scss

```diff
@@ -25,6 +25,14 @@
     margin-bottom: 20px;
   }
 
+  &--logo {
+    width: 75px;
+
+    @media (min-width: 430px) {
+      width: 107px;
+    }
+  }
+
   &__products {
```

Comment:
this fix gives `cards--logo` a fixed width and is especially helpful when there is only one card to show (as in the case with "farmers" theme).
![image](https://user-images.githubusercontent.com/63985172/133693778-ea58d27f-48c5-4eb8-b0f3-f20386ad10e4.png)